packages/math-new/src/index.ts

```diff
@@ -41,19 +41,19 @@ export class AMM {
     switch (amm.poolType) {
       case "MetaStable":
         this.math = new ExtendedMetaStableMath(
-          amm.poolParams as IMetaStableMath,
+          amm.poolParams as IMetaStableMath
         );
       case "MetaStable_2":
```

Comment:
`#comment` "MetaStable_2" is just to serve as a placeholder so that Lu can start working on the Gyro stuff.